python/ingestion/gcs_to_bq_util.py

```diff
@@ -39,8 +39,9 @@ def append_dataframe_to_bq(
   # Repeated fields are not supported with bigquery.Client.load_table_from_dataframe()
   # (See https://github.com/googleapis/python-bigquery/issues/19). We have to use
   # load_table_from_json as a workaround.
-  load_job = client.load_table_from_json(
-      frame.to_dict(orient='records'), table_id, job_config=job_config)
+  result = frame.to_json(orient='records')	
+  json_data = json.loads(result)	
+  load_job = client.load_table_from_json(json_data,	table_id,job_config=job_config)
```

Comment:
Did the to_dict way have some edge case or something?

Reply:
There were parse errors for the covid data source. But I was able to get this way working for all the data sources (minus primary care which has some other issue).